server/api/outreach/outreach.model.js

```diff
@@ -18,4 +18,4 @@ var InterventionSchema = new Schema({
   }]
 });
 
-module.exports = mongoose.model('Outreach', InterventionSchema);
+module.exports = mongoose.model('Outreach', OutreachSchema);
```

Comment:
I think I might have made this change already on my most recent pull request? Not that big of an issue, should be a minor merge fix if true.